### ISSUE TITLE
Database written to disk when application minimized to tray

### ISSUE THREAD

**Issue by Hiche3** [QuiteRSS/quiterss#1578]
Database file is written to disk every time when application is minimized to tray, regardless of what the "Save DB stored in memory to file every X minutes" is set at. This can be observed in the debug file.

**Comment by birdie-github** [comment]
A patch for this is here:

https://bugzilla.redhat.com/show_bug.cgi?id=2190301

No idea what was the idea behind that.

**Comment by Hiche3** [comment]
Looks like the patch isn't working.

**Comment by birdie-github** [comment]
> Looks like the patch isn't working.

Yeah, I'm not a programmer, I thought it would be a quick fix, sorry.

**Comment by birdie-github** [comment]
An active fork with full Qt6 support is here: https://github.com/birdie-github/notquiterss

It has this bug fixed.

You're welcome to try it.
